lambda/delete-activity-log/tests/delete-activity-log.test.ts

```diff
@@ -0,0 +1,95 @@
+import "aws-sdk-client-mock-jest";
+import { DynamoDBDocumentClient, DeleteCommand } from "@aws-sdk/lib-dynamodb";
+import { SendMessageCommand, SQSClient } from "@aws-sdk/client-sqs";
+import { mockClient } from "aws-sdk-client-mock";
+
+import { TEST_USER_DATA, TEST_SNS_EVENT } from "./test-helpers";
+import {
+  deleteActivityLog,
+  handler,
+  validateUserData,
+} from "../delete-activity-log";
+
+const dynamoMock = mockClient(DynamoDBDocumentClient);
+const sqsMock = mockClient(SQSClient);
+
+describe("deleteUserData", () => {
+  beforeEach(() => {
+    dynamoMock.reset();
+
+    process.env.TABLE_NAME = "TABLE_NAME";
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("deletes item from DynamoDB", async () => {
+    await deleteActivityLog(TEST_USER_DATA);
+    expect(dynamoMock).toHaveReceivedCommandWith(DeleteCommand, {
+      TableName: process.env.TABLE_NAME,
+      Key: { user_id: TEST_USER_DATA.user_id },
+    });
+  });
+});
+
+describe("handler", () => {
+  beforeEach(() => {
+    dynamoMock.reset();
+    sqsMock.reset();
+    process.env.TABLE_NAME = "TABLE_NAME";
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("it iterates over each record in the batch", async () => {
+    await handler(TEST_SNS_EVENT);
+    expect(dynamoMock.commandCalls(DeleteCommand).length).toEqual(2);
+  });
```

Comment:
Could we add something similar to : https://github.com/alphagov/di-account-management-backend/pull/105/files#diff-fdfb65ad97796cd1e379b0543ddbc9f13e9940177fce4359209d250f5fef7232R29